mist-api/src/lib/RouteHelper.js

```diff
@@ -48,16 +65,26 @@ class RouteHelper {
 	 *
 	 * @returns {Function} - express compatible handler function
 	 */
-	findManyHandler() {
+	findManyHandler(sequelize=null) {
```

Comment:
This may be moot in the long run (due to a different implementation), but no need to pass in `sequelize` here. Please double check, but it should be available from the model like follows: `this.model_.sequelize`.